src/main/java/org/cascadebot/cascadebot/events/CommandListener.java

```diff
@@ -183,6 +184,15 @@ private void processCommands(GuildMessageReceivedEvent event, GuildData guildDat
             }
             dispatchCommand(cmd, context);
         }
+
+        if (guildData.getSettings().willAllowTagCommands()) {
+            if (guildData.getTagInfo().containsKey(trigger)) {
+                Tag tag = guildData.getTag(trigger);
+
+                context.reply(tag.formatTag(context)); //Are we adding perms to tags? If so i need to implement that. - Ryan
```

Comment:
Yes we are adding perms to tags

Reply:
Think we should hold of this pr till we get per guild permissions or just merge it now?

Reply:
Make this comment an actual TODO so we can find it and merge it I think